Folder and filenaming (#15)
inbo/protocols: README.md

The subject-matter specialist will then give sensible default values for these parameters, but these values can be changed in project-specific protocols.

## Folder and filename syntaxis

In this section, we describe the conventions we adhere to for naming of files and folders.

> Each protocol will be placed in a subfolder of the folder corresponding to the theme (for thematic protocols) or project (for project-specific protocols) to which it belongs (see [Repository structure](#repository-structure)). 

Reviewer: ```suggestion
Each protocol is placed in a subfolder of the folder corresponding to the theme (for thematic protocols) or project (for project-specific protocols) to which it belongs (see [Repository structure](#repository-structure)). 
```

The naming syntaxis for the subfolder is `protocol-code_short-title_language-code` and will be automatically generated from input you provide to [protocolhelper functions](#starting-a-new-protocol-with-the-aid-of-protocolhelper-functions).

> Note that this subfolder name is stable for different versions of the same protocol, because the files in these subfolders are subject to `Git` version control. 

Reviewer: ```suggestion
Note that the names of this subfolder and of its files are stable for different versions of the same protocol, because the files in these subfolders are subject to `Git` version control and the version number is kept inside the special file `index.Rmd`. 
```

This has some relation with https://github.com/inbo/protocolhelper/pull/9#discussion_r435954290 as well.

> The subfolder contains `Rmarkdown` (i.e. `.Rmd`) files, a `_bookdown.yml` file and two folders, named `data` and `media`. 

Author: Actually, they are created by default (and can be left empty if not needed).

> The subfolder contains `Rmarkdown` (i.e. `.Rmd`) files, a `_bookdown.yml` file and two folders, named `data` and `media`. 

Reviewer: I understand that `create_sfp()` will do that. However, here we describe the protocols repo. If left empty, those folders will (and can) not be committed and so you won't have (nor need) them in the git repository. Seen from the repo, I think you best call them 'optional' or 'if needed' or something like that.

> The subfolder contains `Rmarkdown` (i.e. `.Rmd`) files, a `_bookdown.yml` file and two folders, named `data` and `media`. 

Author: See 774eba0

Together, these files form a [bookdown book](https://bookdown.org/yihui/bookdown/), i.e. a collection of files to be read in a linear sequence that can be rendered into a 'book' (a protocol in our case) in any of a number of possible formats.

> With each release of a protocol a version number will be added to the metadata in the `index.Rmd` file.
> Apart from this file, other `Rmarkdown` files contain the contents of the individual chapters of a protocol. 
> The naming of these files follows this syntax: `##_chapter-title.Rmd`, where the `##` indicates the chapter number.
> The folders, named `data` and `media` serve to store, respectively, tabular data and graphics files that belong to and are used in the protocol.
> The `_bookdown.yml` also holds metadata information such as the name of the output file and folder (both have the same syntax as before: `protocol-code_short-title_language-code`) to which the rendered version of the protocol will be written.

Reviewer: ```suggestion

- Apart from the `index.Rmd` file, other `Rmarkdown` files contain the contents of the individual chapters of a protocol. 
- The naming of these files follows this syntax: `##_chapter-title.Rmd`, where the `##` indicates the chapter number.
- The folders, named `data` and `media` serve to store, respectively, tabular data and graphics files that belong to and are used in the protocol.
- The `_bookdown.yml` also holds metadata information such as the name of the output file and folder (both have the same syntax as before: `protocol-code_short-title_language-code`) to which the rendered version of the protocol will be written.
```
